tests/evaluation/test_stochastic.py

```diff
@@ -0,0 +1,66 @@
+# This file contains stochastic tests for mechanism and SQL queries that should pass
+# If these tests fail, then one of the 3 promises of DP system - Privacy, Accuracy, Utility are not being met
+
+import sys
+import os
+sys.path.append(os.path.join(os.path.dirname(__file__), '..', '..'))
+import pytest
+from evaluation.DPVerification import DPVerification
+from evaluation.Exploration import Exploration
+from evaluation.Aggregation import Aggregation
+
+dv = DPVerification(dataset_size=10000)
+ag = Aggregation(t=1, repeat_count=10000)
+
+class TestStochastic:
+    def test_dp_predicate_count(self):
+        d1_query = "SELECT COUNT(UserId) AS UserCount FROM d1.d1"
+        d2_query = "SELECT COUNT(UserId) AS UserCount FROM d2.d2"
+        dp_res, acc_res = dv.dp_query_test(d1_query, d2_query, plot=False, repeat_count=10000)
+        print("Result of DP Predicate Test on COUNT Query: ", dp_res)
+        assert(dp_res == True)
+
+    def test_dp_predicate_sum(self):
+        d1_query = "SELECT SUM(Usage) AS TotalUsage FROM d1.d1"
+        d2_query = "SELECT SUM(Usage) AS TotalUsage FROM d2.d2"
+        dp_res, acc_res = dv.dp_query_test(d1_query, d2_query, plot=False, repeat_count=10000)
+        print("Result of DP Predicate Test on SUM Query: ", dp_res)
+        assert(dp_res == True)    
+    
+    def test_dp_predicate_mean(self):
+        d1_query = "SELECT AVG(Usage) AS MeanUsage FROM d1.d1"
+        d2_query = "SELECT AVG(Usage) AS MeanUsage FROM d2.d2"
+        dp_res, acc_res = dv.dp_query_test(d1_query, d2_query, plot=False, repeat_count=10000)
+        print("Result of DP Predicate Test on MEAN Query: ", dp_res)
```

Comment:
we should log these results with the logging library

import logging
test_logger = logging.getLogger(__name__)  # at the top of the file
logging.setLogLevel(logging.DEBUG)  # while writing/debugging the tests

test_logger.debug("message")  # in the tests.

Reply:
Nice. Thanks for the snippet. Used it for logging prints.